src/pythoninterpretermodule.hpp

```diff
@@ -17,15 +17,18 @@
 #include <memory>
 
 #include "module.hpp"
+#include "python/python_backend.hpp"
 
 namespace ovms {
 
 class PythonInterpreterModule : public Module {
+    PythonBackend* pythonBackend;
```

Comment:
```suggestion
    PythonBackend* pythonBackend{nullptr};
```
This way you do not need to do:
```
PythonInterpreterModule::PythonInterpreterModule() {pythonBackend = nullptr;}
```
in cpp unit

Reply:
Makes sense